Ollie Portfolio

olyolyoly opened the pull request.

olyolyoly committed:
creating gitignore

olyolyoly committed:
default html

olyolyoly committed:
please review my code

alicemcneil commented on index.html:
```diff
@@ -0,0 +1,48 @@
+<body>
+    <nav>
+        <div class="menu">
+            <div class="buttons"> HOME </div>
```
could you use` <a>` (anchor/link tags) to make these clickable eventually?

alicemcneil commented on css/portfolio.css:
```diff
@@ -0,0 +1,92 @@
+    font-weight: bold;
+    color: white;
+    font-size: 5rem;
+}
```
section1 - this text looks a little small on the page

alicemcneil commented on css/portfolio.css:
```diff
@@ -0,0 +1,92 @@
+/*this scales fonts when zoom is altered in OS*/
+
+nav {
+    background-color: #DCDCDC;
```
does this grey//white contrast stand out enough?

alicemcneil commented on index.html:
```diff
@@ -0,0 +1,48 @@
+    <section class="section1">jhgcjghcjhg</section>
+
+    <section class="section2">
+        About me
```
i guess 'about me' is a heading? maybe use a `<h3> ` or other heading tag here

alicemcneil commented on css/portfolio.css:
```diff
@@ -0,0 +1,92 @@
+
+}
+.menu {
+}
```
`letter-spacing: 5px;` this is a way to make space between your letters, might help to make your buttons more readable even when you decrease the size

alicemcneil commented on css/portfolio.css:
```diff
@@ -0,0 +1,92 @@
+* {
```
i really like this design and font choice! :)

alicemcneil commented on index.html:
```diff
@@ -0,0 +1,48 @@
+
+
+
+</html>
```
well done, this is really good! proud of you :)

olyolyoly committed:
please check my code
I have updated placeholders for portfolio
I have added css hover to nav buttons
